src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -235,6 +246,8 @@ private void MethodChecks(MethodDeclarationSyntax syntax, Binder withTypeParamsB
                 else if (!IsStatic)
                 {
                     diagnostics.Add(ErrorCode.ERR_BadExtensionMeth, location);
+                    // PROTOTYPE(readonly-ref): to match PEMethodSymbol, should we SetIsExtensionMethod(false)? See #20282 for more details
```

Comment:
I want to keep it until we decide on the best way to handle the differences between methods and PE symbols?

---
In reply to: [122884455](https://github.com/dotnet/roslyn/pull/19703#discussion_r122884455) [](ancestors = 122884455)